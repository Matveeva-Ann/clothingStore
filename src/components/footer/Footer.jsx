import React from 'react';
import Company from "./Company";
import { FooterLinks, FooterWrapper } from "./Footer.styled";
import Location from "./Location";
import MoreInfo from "./MoreInfo";
import NeedHelp from "./NeedHelp";


export default function Footer() {

  return (
    <FooterWrapper data-testid="footer-wrapper">
      <FooterLinks>
        <NeedHelp></NeedHelp>
        <Company></Company>
        <MoreInfo></MoreInfo>
        <Location></Location>
      </FooterLinks>

    </FooterWrapper>
  )
}