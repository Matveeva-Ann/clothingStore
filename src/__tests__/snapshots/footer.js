import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Footer from '../../components/footer/Footer.jsx';

test('Footer renders correctly', () => {
  const { getByTestId, getByText } = render(<Footer />);
  expect(getByTestId('footer-wrapper')).toBeInTheDocument();
  expect(getByText('Need Help')).toBeInTheDocument();
  expect(getByText('Company')).toBeInTheDocument();
  expect(getByText('More Info')).toBeInTheDocument();
  expect(getByText('Location')).toBeInTheDocument();

  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
