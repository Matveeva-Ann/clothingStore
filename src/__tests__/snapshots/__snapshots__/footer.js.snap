// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="css-88d1mg"
  data-testid="footer-wrapper"
>
  <div
    className="css-qk3z1t"
  >
    <div
      className="css-10n2x14"
    >
      <h3
        className="css-1o1a0c3"
      >
        Need Help
      </h3>
      <ul
        className="css-rj3s1a"
      >
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            Contact Us
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            Track Order
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            Returns & Refunds
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            FAQ's
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            Career
          </a>
        </li>
      </ul>
    </div>
    <div
      className="css-10n2x14"
    >
      <h3
        className="css-1o1a0c3"
      >
        Company
      </h3>
      <ul
        className="css-rj3s1a"
      >
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            About Us
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            euphoria Blog
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            euphoriastan
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            Collaboration
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
            href=""
          >
            Media
          </a>
        </li>
      </ul>
    </div>
    <div
      className="css-10n2x14"
    >
      <h3
        className="css-1o1a0c3"
      >
        More Info
      </h3>
      <ul
        className="css-rj3s1a"
      >
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            support@euphoria.in
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            Term and Conditions
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            Privacy Policy
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            Shipping Policy
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            Sitemap
          </a>
        </li>
      </ul>
    </div>
    <div
      className="css-10n2x14"
    >
      <h3
        className="css-1o1a0c3"
      >
        Location
      </h3>
      <ul
        className="css-rj3s1a"
      >
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            support@euphoria.in
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            Eklingpura Chouraha, Ahmedabad Main Road
          </a>
        </li>
        <li
          className="css-ebo1y3"
        >
          <a
            className="css-1gb7hpx"
          >
            (NH 8- Near Mahadev Hotel) Udaipur, India- 313002
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
